Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -414,21 +425,23 @@ public bool OpenSolution(string solutionFolder)
                 solution.SetReportsConfigurations();
                 Solution = solution;                
                 UserProfile.LoadRecentAppAgentMapping();
-                
+                Console.WriteLine("loaded Agent Mapping");
+
                 if (!RunningInExecutionMode)
                 {
                     AppSolutionRecover.DoSolutionAutoSaveAndRecover();   
                 }
 
+                Console.WriteLine("Autosave");
                 //Solution items upgrade
                 SolutionUpgrade.CheckSolutionItemsUpgrade(solutionFolder, solution.Name, solutionFiles.ToList());
-                
+
                 // No need to add solution to recent if running from CLI
                 if (!RunningInExecutionMode)  
                 {
                     UserProfile.AddSolutionToRecent(solution);
                 }
-
+                Console.WriteLine("solution added to recent");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)